src/frontEnd/nav/SideNav.tsx

```diff
@@ -40,41 +41,44 @@ export const SideNavUnstyled = ({navTo, isSideNavOpen, isSideNavExpanded, onSide
             </ListItem>
             <Collapse in={isSideNavExpanded} timeout="auto" unmountOnExit>
             <List component="div" disablePadding>
-             <ListItem button className={classes.nested} onClick={() => navTo("/organizations/new")}>
+             <ListItem button className={classes.nested} onClick={() => onSideNavCollapseRequested("/organizations/new")}>
               <ListItemIcon>
                 <AddCircleOutline />
               </ListItemIcon>
               <ListItemText primary="Create New" />
              </ListItem>
-             <ListItem button className={classes.nested} onClick={() => navTo("/organizations/search")}>
+             <ListItem button className={classes.nested} onClick={() => onSideNavCollapseRequested("/organizations/search")}>
               <ListItemIcon>
                 <Search />
               </ListItemIcon>
               <ListItemText primary="Search" />
              </ListItem>
             </List>
             </Collapse>
-            <ListItem button onClick={() => navTo("/volunteering/search")}>
+            <ListItem button onClick={() => onSideNavCollapseRequested("/volunteering/search")}>
               <ListItemIcon><PanTool /></ListItemIcon>
               <ListItemText primary="Volunteering" />
             </ListItem>
-            <ListItem button onClick={() => navTo("/") }>
+            <ListItem button onClick={() => onSideNavCollapseRequested("/") }>
               <ListItemIcon><Favorite /></ListItemIcon>
               <ListItemText primary="My Heart Collection" />
             </ListItem>
-            <ListItem button onClick={() => navTo("/castleRisk") }>
+            <ListItem button onClick={() => onSideNavCollapseRequested("/castleRisk") }>
               <ListItemIcon><DonutSmallIcon /></ListItemIcon>
               <ListItemText primary="Castle Risk" />
             </ListItem>
-            <ListItem button onClick={() => navTo("/rest-guru") }>
+            <ListItem button onClick={() => onSideNavCollapseRequested("/rest-guru") }>
               <ListItemIcon><DonutSmallIcon /></ListItemIcon>
               <ListItemText primary="rest.guru" />
             </ListItem>
           </List>
     </Drawer>
 }
+const largestPhoneWidth = 480;
 
-const drawerWidth = 240;
+const drawerWidth = window.innerWidth <= largestPhoneWidth
```

Comment:
wondering how this works when u resize down to mobile size but don't refresh the screen, will it still use whatever it was when the page was loaded?

we can always make `const drawerWidth = () => window.innerWidth ...` and then it will recompute it, so that if we do change the viewsize without refreshing, it will redraw and recompute this condition... i believe we already handle when the window resizes to tell it to redraw, if not we can add that as well...

Reply:
refreshing is required as it stands right now. thanks for the insight

Reply:
wow that's way better! Thank you!